.github/workflows/multus.yaml

```diff
@@ -110,3 +110,62 @@ jobs:
         # independent from other tests as long as nodes are tainted and labeled
         if: steps.taint.outcome == 'success' && !cancelled()
         run: ./tests/scripts/multus/test-240-stretch-cluster-only.sh
+
+
+  test-rook-multus-install:
```

Comment:
Is this a temporary change?

Reply:
I probably should move it to the canary test when I'm done

Reply:
also, I can I can close this PR https://github.com/rook/rook/pull/14159 as you are adding the tests here